src/test/java/teammates/test/cases/browsertests/InstructorFeedbackSubmitPageUiTest.java

```diff
@@ -722,52 +791,71 @@ private void testConstSumSubmitAction() {
+        submitPage.fillResponseTextBox(qnNumber, 0, 2, "25");
         assertEquals("Actual total is 75! Distribute the remaining 25 points.\n"
-                     + "All options are given 25 points. "
-                     + "Please allocate different points to at least one option.",
-                     submitPage.getConstSumMessage(qnNumber, 0));
-        submitPage.fillResponseTextBox(qnNumber, 0, 0, "60");
-        assertEquals("Actual total is 110! Remove the extra 10 points allocated.\n"
+                        + "All options are given 25 points. "
+                        + "Please allocate different points to at least one option.",
+                submitPage.getConstSumMessage(qnNumber, 0));
+
+        submitPage.fillResponseTextBox(qnNumber, 0, 1, "40");
+        assertEquals("Actual total is 90! Distribute the remaining 10 points.\n"
                      + "At least one option has been allocated different number of points.",
                      submitPage.getConstSumMessage(qnNumber, 0));
-        submitPage.fillResponseTextBox(qnNumber, 0, 1, "60");
-        submitPage.fillResponseTextBox(qnNumber, 0, 2, "60");
-        assertEquals("Actual total is 180! Remove the extra 80 points allocated.\n"
-                     + "All options are given 60 points. "
-                     + "Please allocate different points to at least one option.",
-                     submitPage.getConstSumMessage(qnNumber, 0));
 
-        qnNumber = 24;
-        submitPage.fillResponseTextBox(24, 0, 0, "33");
-        submitPage.fillResponseTextBox(24, 0, 1, "33");
-        submitPage.fillResponseTextBox(24, 0, 2, "33");
-        assertEquals("Actual total is 99! Distribute the remaining 1 points.\n"
-                     + "All options are given 33 points. "
-                     + "Please allocate different points to at least one option.",
-                     submitPage.getConstSumMessage(qnNumber, 0));
-        submitPage.fillResponseTextBox(24, 0, 0, "31");
-        assertEquals("Actual total is 97! Distribute the remaining 3 points.\n"
+        submitPage.fillResponseTextBox(qnNumber, 0, 0, "40");
+        submitPage.fillResponseTextBox(qnNumber, 0, 2, "40");
+        assertEquals("Actual total is 120! Remove the extra 20 points allocated.\n"
+                        + "All options are given 40 points. "
+                        + "Please allocate different points to at least one option.",
+                submitPage.getConstSumMessage(qnNumber, 0));
+
+        submitPage.fillResponseTextBox(qnNumber, 0, 0, "30");
+        assertEquals("Actual total is 110! Remove the extra 10 points allocated.\n"
                      + "At least one option has been allocated different number of points.",
                      submitPage.getConstSumMessage(qnNumber, 0));
-        submitPage.fillResponseTextBox(24, 0, 2, "36");
+
+        submitPage.submitWithoutConfirmationEmail();
+        submitPage.waitForTextsForAllStatusMessagesToUserEquals(
```

Comment:
The validation comes from frontend rather than backend (so you are testing frontend rather than backend). 

This is also the reason why the test should be written in action level.